src/FormBuilder/resources/js/form-builder/src/blocks/fields/donor-name/Edit.tsx

```diff
@@ -84,15 +97,12 @@ export default function Edit({
                                 />
                             </div>
                             {!!showHonorific && (
-                                <FormTokenField
-                                    tokenizeOnSpace={true}
-                                    label={__('Title Prefixes', 'give')}
-                                    value={honorifics}
-                                    suggestions={['Mr', 'Ms', 'Mrs']}
-                                    // placeholder={__('Select some options', 'give')}
-                                    onChange={(tokens) => setAttributes({honorifics: tokens})}
-                                    displayTransform={titleLabelTransform}
-                                    saveTransform={titleValueTransform}
+                                <Options
```

Comment:
Should we be supporting the _selectable_ feature of the options component? With this implementation there is extra UI to select an option that is not used.

Looking at the FFM Dropdown block the _selected_ option is reflected in the select input of the field.

Reply:
@kjohnson resolved https://github.com/impress-org/givewp/pull/7003/commits/7955f63800376ea4f2d8236282d6aead52b2f9dc

I added a `selectable` parameter to the `Options` component. Now UI for selecting the default option is optional. The parameter default value is set to `true` for backward compatibility.